app/views/admin/failed_bikes/show.html.haml

```diff
@@ -1,30 +1,40 @@
-%p
-  ID:
-  = @b_param.id
-%p
-  Creator:
-  - if @b_param.creator.present?
-    = @b_param.creator.email
-  - else
-    No creator
-%p
-  Created:
-  = @b_param.created_at.strftime("%-I:%M %p on %-m/%-d/%Y")
-%p
-  Modified:
-  = @b_param.updated_at.strftime("%-I:%M %p on %-m/%-d/%Y")
-
-
-
-
-%h3.padded
+%table.table-list
+  %tbody
+    %tr
+      %td
+        ID
+      %td
+        = @b_param.id
+    %tr
+      %td
+        Creator
+      %td
+        -if @b_param.creator.present?
+          = @b_param.creator.email
+        -else
+          No Creator
+    %tr
+      %td
+        created_at
+      %td.convertTime
```

Comment:
I think these should be `preciseTime`. 

In general, on show pages, things should show precise time because we don't need to conserve space.

Reply:
Got it. I'll go over this one and paints and see where this should be added.

Reply:
Great!